docs/learn/learn-account-abstraction.md

```diff
@@ -0,0 +1,114 @@
+
+While the account’s private keys grant users control and ownership, losing them results in losing
+access to digital assets and fragmentation of your digital identity since you will need to create a
+new account with a new set of keys. This poses a hurdle for both users and developers regarding
+security and adoption.
+
+## Definition of Account Abstraction
+
+The term _abstraction_ refers to separating the user experience from the private key, enabling a
+piece of code to dictate account behavior. This allows for increased flexibility of accounts that
+originally were not engineered to be flexible and decreased chances of key mismanagement.
+
+Users are still responsible for their keys, but through account abstraction, they can take
+precautions to ensure they do not end up losing their accounts.
+
+Account abstraction introduces a layer of on-chain logic that controls an account, typically in the
+form of a smart contract. Without a smart contract, abstracting accounts would require changes in
+the core architecture of the protocol.
+
+The highly generic codebase of Polkadot's architecture makes accounts natively flexible and abstract
+without the direct need for smart contracts.
+
+## Origin Abstraction in Polkadot
+
+Adopting a generic design is crucial in scaling Web3 technologies. Abstraction and generalization of
+protocols are essential to improving user experience and security in blockchain adoption.
+
+When users interact with a blockchain they call _dispatchable_ functions to do something. Because
+those functions are called from the outside of the blockchain interface, in Polkadot's terms any
+action that involves a dispatchable function is an [extrinsic](./learn-extrinsics.md). Extrinsics
+are calls coming from the _outside_ of the blockchain interface that (if successfully executed)
+invoke some changes in the _inside_ of a blockchain, the blockchain's state. An extrinsic is always
+directed to a specific function within a particular pallet (or module).
+
+For example, the `balances.transferKeepAlive` extrinsic is directed to the `transferKeepAlive`
+function within the `balances` pallet. If successful, the execution of that function will transfer
+funds between two accounts, changing the balances of those accounts and thus the chain state (as
+accounts hold some state within the blockchain).
+
+In [Substrate](../general/glossary.md#substrate)'s FRAME, functions are not necessarily called by
+accounts. Functions can be called by any origin, where origins are caller-personas associated with
+privilege levels. For example, the Polkadot [OpenGov](./learn-polkadot-opengov.md) has different
+origins with different privileges, such as allocating treasury funds, cancelling a referendum, etc.
+Neither of those origins is subservient to the concept of an account or assume anything about state
+or associated data. Custom origins can be created while designing your chain using the
+[Polkadot SDK](https://github.com/paritytech/polkadot-sdk).
```

Comment:
We mention Polkadot SDK here, but substrate somewhere else.  It's not a huge deal but it should probably be consistent